src/context/Localization/language/en.json

```diff
@@ -1145,11 +1145,12 @@
     "masterKeyParagraph3": "Simply backing up or noting down the Master Recovery Key gives you access to all this data which is updated automatically.",
     "masterKeyNote": "Master Recovery Key also gives access to the app’s hot keys. Please secure the key properly.",
     "personalCloudTitle": "Personal Cloud Backup",
-    "personalCloudDescpMain": "Use your",
-    "personalCloudParagph1": "Wallet configuration files for vaults can alternatively be stored on a personal cloud. This ensures that you or your heir have access to them independent of the Keeper app.",
-    "personalCloudParagph2": "These files are also encrypted with the Master Recovery Key which needs to be backed up properly.",
-    "personalCloudNotes": "This action needs to be performed regularly as configuration details change",
+    "personalCloudDescpMain": "This action needs to be performed regularly as",
```

Comment:
This text doesn't seem right

Reply:
It is according to the design.

Reply:
It's an incomplete and outdated sentence. We should remove it

Reply:
done